taxi_service/wsgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+WSGI config for taxi_service project.
+
+It exposes the WSGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/howto/deployment/wsgi/
+"""
+
+import os
+
+from django.core.wsgi import get_wsgi_application
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "taxi_service.settings")
+
+application = get_wsgi_application()
```

Comment:
Add a `requirements.txt` file to the project where you will put the output of the `pip freeze`.